docs/components/RPA/production.md

```diff
@@ -0,0 +1,47 @@
+---
+id: production
+title: Setting up RPA for production deployments
+---
+
+This article covers the specific configuration of your RPA runner. For the basic getting started guide, check the
+[getting started guide](./getting-started.md).
+
+## Using Secrets
```

Comment:
Secrets are disabled by default, and are enabled by configuring the property `camunda.rpa.zeebe.secrets.secrets-endpoint` (the Worker has no way of determining this). 

It doesn't apply to the prod SaaS environment (and therefore might not be worth mentioning), but the default value for `camunda.rpa.zeebe.secrets.token-audience` only works for prod, and must be configured for the dev instances.

Reply:
I'd like to avoid exposing the `cluster-api.cloud.camunda.io` endpoint to the users, as the console API is internal and [not publicly documented](https://docs.camunda.io/docs/next/apis-tools/camunda-api-rest/camunda-api-rest-overview/)

Reply:
I added the config option in [321d9d5](https://github.com/camunda/camunda-docs/pull/5186/commits/321d9d514d7bb1c95487845a686f5c575e9e0922) for now